Documentation/General/finalReport/appendix-neuralnetwork.tex

```diff
@@ -0,0 +1,41 @@
+\chapter{Neural Networks}
+
+One of the bigger issues we had to deal with in MoodCat was to classify songs
+to moods. The initial plan was to create a neural network that could do this.
+
+Some of the MoodCat team members have followed a course about computational intelligence,
+in which different ways to use learning algorithms to solve different kinds of problems.
+One widely used technique to classify objects by using the objects' features as inputs
```

Comment:
One widely used technique to classify objects is by using the objects' features as inputs for the neural network.